libraries/helpers.rb

```diff
@@ -0,0 +1,33 @@
+module OSLResources
+  module Cookbook
+    module Helpers
+      # osl_ifconfig helpers
+      def default_nm_controlled
+        node['platform_version'].to_i >= 8 ? 'yes' : 'no'
+      end
```

Comment:
Do we need to worry about this being checked on Debian?

Reply:
The network related resources are only for centos